smithy-model/src/main/java/software/amazon/smithy/model/validation/validators/TargetValidator.java

```diff
@@ -204,7 +204,7 @@ private void validateDeprecatedTargets(
         deprecatedTrait.getMessage().ifPresent(message -> builder.append(". ").append(message));
         deprecatedTrait.getSince().ifPresent(since -> builder.append(" (since ").append(since).append(')'));
         events.add(ValidationEvent.builder()
-                           .id("DeprecatedShape")
+                           .id(String.format("DeprecatedShape.%s", target.getId()))
```

Comment:
Use string concatenation instead, e.g., `"DeprecatedShape" + target.getId()`

```suggestion
                           .id("DeprecatedShape." + target.getId())
```